synthtool/gcp/common.py

```diff
@@ -105,6 +105,7 @@ def py_library(self, **kwargs) -> Path:
         if "samples" not in kwargs:
             self.excludes += ["samples/AUTHORING_GUIDE.md", "samples/CONTRIBUTING.md"]
 
+        self.excludes += [".kokoro/docker/docs/fetch_gpg_keys.sh"]
```

Comment:
If you add it to the excludes list the file will not be copied over. I think escaping the comment string should work? https://jinja.palletsprojects.com/en/2.11.x/templates/#escaping

Reply:
Ya, I used `raw`. It looks fine.